inc/Engine/Optimization/RUCSS/Database/Tables/UsedCSS.php

```diff
@@ -80,4 +80,25 @@ public function delete_old_used_css() {
 		return $rows_affected;
 	}
 
+	/**
+	 * Get all used_css which were not accessed in the last month.
+	 *
+	 * @return array
+	 */
+	public function get_old_used_css() : array {
+		// Get the database interface.
+		$db = $this->get_db();
+
+		// Bail if no database interface is available.
+		if ( empty( $db ) ) {
+			return false;
+		}
+
+		$prefixed_table_name = $this->apply_prefix( $this->table_name );
+		$query               = "SELECT * FROM `$prefixed_table_name` WHERE `last_accessed` <= date_sub(now(), interval 1 month)";
```

Comment:
what do u think if we selected only `ID` and at cron we can get the full row not with instantiated `UsedCSS_Row` but with  getting by the ID?

Reply:
Not sure if this will be ok. Because there will be another DB SELECT to grab each UsedCSS_Row element. 
Currently, for this operation there will be only 1 DB select. With grabing a single id and then for each ids to fetch each element, will make too many DB calls.

Reply:
Ok, we may enhance this query one day to use BerlinDB query and return rows directly.
After a talk with Cristina, it's hard to be done in such a way.